udata_hydra/cli.py

```diff
@@ -212,6 +225,35 @@ async def purge_checks(limit=2):
         await context["conn"].execute(q, resource_id, limit)
 
 
+@cli
+async def purge_csv_tables():
+    """Delete converted CSV tables for resources no longer in catalog"""
+    q = """
+        SELECT parsing_table FROM csv_analysis
+        WHERE parsing_table IN (
+            SELECT md5(url) FROM catalog WHERE deleted = TRUE
```

Comment:
When loading catalog:

`ON CONFLICT (dataset_id, resource_id, url) DO UPDATE SET deleted = FALSE`

So for a given `resource_id`, if an `url` has changed, it will be marked as `deleted` (not included in the CONFLICT triplet) and caught by `SELECT md5(url) FROM catalog WHERE deleted = TRUE`. Right?